benchmarks/shelley3pools/fund-addresses-generic.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
```

Comment:
why is this file "generic"? how does it differ from the normal version?

Reply:
We fork this script N times and each script is separated to its own directory to avoid conflicting files etc. For example, `cat ${WORKDIR}/Test_${ID}/txs` where `Test_${ID}` is the ID of the script.